db/migrate/20170703094235_add_hash_expression_to_miq_alert.rb

```diff
@@ -0,0 +1,35 @@
+class AddHashExpressionToMiqAlert < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base
+    serialize :expression
+    serialize :miq_expression
```

Comment:
Actually I just realized this is a problem.  If the MiqExpression contains any object references, then those will be materialized on deserialization.  I'm not sure what's actually in an MiqExpression, but if it's a class name, and in the future that class no longer exists, then this migration will fail on db:migrate because it will be trying to load that actual class from app/models via autoload.

Reply:
cc @imtayadeway @gtanzillo @jrafanie Keep me honest here.  I know in other migrations I've avoided deserializing MiqExpressions because of this exact reason.

Reply:
@Fryguy Afaik, there are not object references in `MiqExpression` instances. All of the operators are strings and all the operands are also strings or numerics. I also defer to @imtayadeway and @jrafanie to keep me honest. But, even so, would it be safer to treat the `expression` column as a string for the migration to avoid any deserialization altogether?